builtin/mainmenu/tab_settings.lua

```diff
@@ -71,6 +71,24 @@ local function video_driver_fname_to_name(selected_driver)
 	return ""
 end
 
+local default_label = fgettext("Default")
```

Comment:
`fgettext("Default")` can be translated if used directly, but in this case it's needed in multiple cases so it's ok.